<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Cadastro</class>
 <widget class="QMainWindow" name="Form_Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>531</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(29, 92, 132);
border-radius:20px;</string>
    </property>
    <widget class="QLineEdit" name="cadastro_password">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(62, 62, 62);
border: 1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 155, 255);
padding-bottom:7px</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Digite a senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cadastro_usuario">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(62, 62, 62);
border: 1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 155, 255);
padding-bottom:7px</string>
     </property>
     <property name="placeholderText">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: 0</string>
     </property>
     <property name="text">
      <string>ÓÅñ</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cadastrarButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton#cadastrarButton{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
}

QPushButton#cadastrarButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(66, 199, 255, 100), stop:1 rgba(255, 255, 255, 100));
}

QPushButton#cadastrarButton{
background-color:rgba(2, 65, 118, 255);
}
</string>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="repeat_password">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(62, 62, 62);
border: 1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:rgb(255, 155, 255);
padding-bottom:7px</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Repita a senha</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg_label2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
